website/src/runner/index.ts

```diff
@@ -5,7 +5,6 @@
 
 import { loadMonaco } from "../monaco-loader";
 import { IMessageFromRunner, IMessageToRunner, IPreviewState } from "../shared";
-import "./style.scss";
```

Comment:
Where does the runner (which is the code that runs in the playground iframe) include the style now?